astropy/units/equivalencies.py

```diff
@@ -508,6 +508,59 @@ def convert_K_to_Jy(x_K):
     return [(astrophys.Jy, si.K, convert_Jy_to_K, convert_K_to_Jy)]
 
 
+def thermodynamic_temperature(disp, T_cmb=2.7255*si.K):
+    r"""Defines the conversion between Jy/beam and "thermodynamic temperature",
+    :math:`T_{CMB}`, in Kelvins.  The thermodynamic temperature is a unit very
+    commonly used in cosmology.  See, e.g., "Planck 2013 results. IX. HFI spectral response"
+    (Planck Collaboration 2013) eqn 8 (available on `arXiv <https://arxiv.org/abs/1303.5070>`__).
```

Comment:
Citation should be in "References". For example, see https://github.com/astropy/astropy/blob/master/astropy/stats/bayesian_blocks.py.